build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.10"
-        classpath "com.android.tools.build:gradle:4.1.1"
+        classpath 'com.android.tools.build:gradle:4.2.0-alpha16'
```

Comment:
You probably need to use Android Studio 4.2 Preview in order to use Jetpack Compose